enacts/crop_suitability/layout_crop_suit.py

```diff
@@ -0,0 +1,337 @@
+                            ],
+                        ),
+                    ),
+                    Block("Pick a point",
+                        dbc.Row(
+                            [
+                                dbc.Col(
+                                    dbc.FormFloating([dbc.Input(
+                                        id = "lat_input",
+                                        min=lat_min,
+                                        max=lat_max,
+                                        type="number",
+                                    ),
+                                    dbc.Label("Latitude", style={"font-size": "80%"}),
+                                    dbc.Tooltip(
+                                        f"{lat_label}",
+                                        target="lat_input",
+                                        className="tooltiptext",
+                                    )]),
+                                ),
+                                dbc.Col(
+                                    dbc.FormFloating([dbc.Input(
+                                        id = "lng_input",
+                                        min=lon_min,
+                                        max=lon_max,
+                                        type="number",
+                                    ),
+                                    dbc.Label("Longitude", style={"font-size": "80%"}),
+                                    dbc.Tooltip(
+                                        f"{lon_label}",
+                                        target="lng_input",
+                                        className="tooltiptext",
+                                    )]),
+                                ),
+                                dbc.Button(id="submit_lat_lng", outline=True, color="primary", children='Submit lat lng'),
+                            ],
+                        ),
+                        "Choose a target year:",
+                        dbc.Input(
+                            id = "target_year",
+                            type = "number",
+                            min = 1961,
+                            max = 2018,
+                            value = CONFIG["param_defaults"]["target_year"],
+                        ),
+                        "Choose a target season:",
+                        dbc.Select(
+                            id="target_season",
+                            value= CONFIG["param_defaults"]["target_season"],
+                            options=[
+                                {"label":"Dec-Feb", "value":"DJF"},
+                                {"label":"Mar-May", "value":"MAM"},
+                                {"label":"Jun-Aug", "value":"JJA"},
+                                {"label":"Sep-Nov", "value":"SON"},
+                            ],
+                        ),
+                    ),
+                    Block(
+                        "Rainfall tolerance",
```

Comment:
I think I put this because Tufa uses "maximum/minimum temperature tolerance" when he emailed me parameters. But looking at the emails I see he does not use that wording for rainfall, he said instead "optimum seasonal total rainfall." So I can put instead "rain suitability" if that is better, or adopt his wording?

Reply:
Good idea to use the terms from the person who's asking for the product then. Unless they're blatantly wrong.